staging/src/k8s.io/legacy-cloud-providers/azure/clients/loadbalancerclient/azure_loadbalancerclient_test.go

```diff
@@ -172,8 +172,10 @@ func TestListWithNextPage(t *testing.T) {
 	resourceID := "/subscriptions/subscriptionID/resourceGroups/rg/providers/Microsoft.Network/loadBalancers"
 	armClient := mockarmclient.NewMockInterface(ctrl)
 	lbList := []network.LoadBalancer{getTestLoadBalancer("lb1"), getTestLoadBalancer("lb2"), getTestLoadBalancer("lb3")}
-	partialResponse, err := json.Marshal(network.LoadBalancerListResult{Value: &lbList, NextLink: to.StringPtr("nextLink")})
+	// LoadBalancerListResult.MarshalJson() doesn't include "nextLink" in its result, hence partialResponse is composed manually below.
+	mcListBytes, err := json.Marshal(lbList)
 	assert.NoError(t, err)
+	partialResponse := []byte(fmt.Sprintf(`{"value": %s, "nextLink": "nextLink"}`, string(mcListBytes)))
```

Comment:
don't compose JSON content using Sprintf. xref https://github.com/kubernetes/kubernetes/issues/81134

comment applies to all three places this is being done

Reply:
thanks for the suggestions, would update per the suggestions.